.github/workflows/php-coding-standards.yml

```diff
@@ -37,4 +39,4 @@ jobs:
         run: vendor/bin/phpcs -q -n --report=checkstyle | cs2pr
 
       - name: PHP Syntax checker
-        uses: StephaneBour/actions-php-lint@7.2
\ No newline at end of file
+        uses: StephaneBour/actions-php-lint@8.0
```

Comment:
I'm thinking we should add a line break so that we don't see the "missing end of line" warning.

Reference: 

- [Why should text files end with a newline?](https://stackoverflow.com/q/729692/379641)

Reply:
Made the changes in [07d3fe4](https://github.com/woocommerce/facebook-for-woocommerce/pull/2624/commits/07d3fe4f5ddfaba0b99825ab4799728177f812be) commit.